PythonTool/TagExpansion.py

```diff
@@ -180,25 +199,12 @@ def expand_vector_tags(mesh_file, element_type, main_dir_name = None):
        LookupError: If the file does not contain any vector tags.
     """
 
-    # Create a dictionary with the MB element types.
-    elements = {
-        "vertex" : types.MBVERTEX, "edge" : types.MBEDGE, "tri" : types.MBTRI,
-        "quad" : types.MBQUAD, "polygon" : types.MBPOLYGON, "tet" : types.MBTET,
-        "pyramid" : types.MBPYRAMID, "prism" : types.MBPRISM, "knife" : types.MBKNIFE,
-        "hex" : types.MBHEX, "polyhedron" : types.MBPOLYHEDRON, "maxtype" : types.MBMAXTYPE,
-        "entityset" : types.MBENTITYSET
-        }
-
     # Load the mesh file to create a reference mesh.
     mb_ref = core.Core()
     mb_ref.load_file(mesh_file)
 
     # Ensure the MB element type is valid.
-    mb_type = elements.get(element_type.lower())
-
-    if mb_type is None:
-        print("WARNING: Please choose a valid MB element type.")
-        exit()
+    mb_type = elements[element_type.lower()]
```

Comment:
Oops - this won't work because I flipped the keys/values relative to what you did....

Reply:
I will merge this and then just go in and flip them back to avoid the need for the reversed dictionary. Thank you!